cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -224,13 +224,17 @@ func (uc *UpstreamController) updatePodStatus() {
 		case msg := <-uc.podStatusChan:
 			klog.Infof("message: %s, operation is: %s, and resource is: %s", msg.GetID(), msg.GetOperation(), msg.GetResource())
 
-			namespace, podStatuses := uc.unmarshalPodStatusMessage(msg)
+			namespace, nodeID, podStatuses := uc.unmarshalPodStatusMessage(msg)
```

Comment:
sorry for the delay, sounds good, how about nodename instead of nodeID here?

Reply:
fixed

Reply:
Thanks, could you please squash the commits?

Reply:
done

Reply:
Using `git rebase` command to squash the 3 commits to 1.